MTM-Frontend/src/components/Accordions/ContactUsAccordion.tsx

```diff
@@ -26,18 +83,71 @@ const ContactUsAccordion: React.FC<ContactUsAccordionProps> = ({
         <AccordionSummary
           aria-controls="contact-panel-content"
           id="contact-header"
-          customStyles={{ backgroundColor: "#F6FBF5" }}
+          customStyles={{ backgroundColor: "#FFFFFF" }}
         >
           <Typography className="square-button">CONTACT & HOURS</Typography>
         </AccordionSummary>
-        <AccordionDetails customStyles={{ backgroundColor: "#F6FBF5" }}>
-          <Typography>
-            For almost two decades, Nashville-based Mother To Mother has been
-            working to ensure children in our community have access to three
-            basic essentials that often slip through the cracks of government
-            subsidies and charitable support:​ diapers car seats a safe place to
-            sleep
-          </Typography>
+        <AccordionDetails customStyles={{ backgroundColor: "#FFFFFF" }}>
+          {/*Contact Us box*/}
+          <Box sx={boxStyle}>
```

Comment:
These boxes can all be componentized to reduce redundant code
<img width="404" alt="image" src="https://github.com/ChangePlusPlusVandy/MotherToMother/assets/81684017/1ff7f20c-db1b-4617-ad1e-978aef81d155">